pywinauto/win32_hooks.py

```diff
@@ -553,7 +553,7 @@ def _kbd_ll_cb(ncode, wparam, lparam):
                 """Forward the hook event to ourselves"""
                 return self._keyboard_low_level_handler(ncode, wparam, lparam)
 
-            self.keyboard_id = windll.user32.SetWindowsHookExW(
+            self.keyboard_id = windll.user32.SetWindowsHookExA(
```

Comment:
Is there any reason for this?

Reply:
Already raised this question. `SetWindowsHookExW` expected unicode strings as arguments, but `GetModuleHandleA` (ascii) passes to the function 3 lines below. One can't be converted directly into another.